schemas/yaml/Language.yml

```diff
@@ -101,11 +101,122 @@ properties:
   dateModified:
     title: 'Date Modified'
     type:  string
-    description: 'The date an optionally time that this Language object was last modified'
+    description: 'The date and optionally time that this Language object was last modified'
     oneOf:
       - format: date
       - format: date-time
 
+  demographics:
+    title:       'Demographics'
+    type:        array
+    uniqueItems: true
+    description: 'An array of Demographic objects, each of which contain demographic information about the Language at a specific point in time, and/or at a specific location'
+    items:
+      title:       'Demographic Data'
+      type:        object
+      description: 'The demographic data of this Language, at a specific point in time, and/or at a specific location'
+
+      properties:
+        date:
+          title:       'Date'
+          type:        string
+          description: 'The date and optionally time that this Demographic data was created'
+          oneOf:
+            - format: date
+            - format: date-time
+
+        location:
+          title:       'Location'
+          type:        object
+          $ref:        'http://schemas.digitallinguistics.io/Location.json'
+          description: 'The geographic Location of this Demographic data'
+
+        notes:
+          title:       'Notes'
+          type:        array
+          uniqueItems: true
+          description: 'A collection of notes about this Demographic data'
+          items:
+            title: 'Note'
+            type:  object
+            $ref:  'http://schemas.digitallinguistics.io/Note.json'
+            description: 'A Note about this Demographic data'
+
+        population:
+          title:       'Population'
+          type:        integer
+          minimum:     0
+          description: 'The Population of speakers included in this Demographic data'
+          
+        status:
+          title:       'Status'
+          type:        object
+          description: 'The Status of this Demographic data on one or more of the following scales: EGIDS, ELCat (using the LEI), GIDS, and/or UNESCO. The official status scales can be found [here](http://langlxmelanesia.com/Paris%20-%20EGIDS%20The%20Expanded%20Graded%20Intergenerational%20Disrupt.pdf).'
+          minimumProperties: 1
+          properties:
+
+            egids:
+              title: 'EGIDS'
+              type:  object
```

Comment:
I hadn't considered this way of structuring the EGIDS field. I was thinking a single String property whose enum values would include _both_ all the labels and the numbers (as Strings).

I like your approach too, though I see one potential issue: Someone could specify a Level and a Label that don't match. If we make this field a single String rather than an Object, this wouldn't be a problem, since the user could only specify one value.

You could also address this problem by using JSON Schema's `oneOf` property though. In this case, EGIDS would still be a String rather than an Object, but its value would _either_ have to be chosen from the list of labels, _or_ from the list of levels (I like your regex for that btw).

I don't think there's a functional difference between the two approaches though. The second would be a little more work for you in creating the schema for it, so I'd suggest doing the first - but I'll leave it up to you!

Reply:
I redefined the property using the `oneOf:` property. Let me know if that is what you had in mind.